### DISCUSSION TITLE
Enabeling vector-logs to listen on host port

### DISCUSSION THREAD

**Discussion by tarbaig** [dokku/dokku#7356] [Q&A]
Hi,
I have log aggregation on dokku enabled via logs-vector and  I would like to send logs from other devices running in the same network to the vector instance.

Is there an intended way of doing this? I can edit the vector.json, and open the port in docker directly, but since dokku manages the ports of all other applications it starts this seems strange.
Also restarting vector-logs resets the config, so I wonder if this is actually discouraged. 

Thanks for your help!

**Comment by josegonzalez** [comment]
Currently, dokku manages the entire config, and it is done in such a way as to only really reference dokku containers. I think at this time, I'm going to hold off on any further enhancements to the vector setup, as while dokku _can_ work with other containers and systems on a host, it's meant to control the entire host. If you're shipping other things, the easiest thing to do is just run vector again elsewhere with a different, targeted config.
